doc/user_guide/transformations.rst

```diff
@@ -396,6 +396,46 @@ Let's plot the results side by side:
     )
     plt.show()
 
+
+Total gradient amplitude (aka Analytic Signal)
+----------------------------------------------
```

Comment:
I would keep the title to "Total gradient amplitude"
```suggestion
Total gradient amplitude
------------------------
```

We can add a admonition for letting users know that this is a.k.a analytic signal:

```rst
.. hint::

    Total gradient amplitude is also known as *analytic signal*.
```

Reply:
Thanks for committing this suggestion. The second hint wasn't part of it, so it wasn't included in that commit. I just pushed it though, so now you can mark this as resolved.